controllers/snapshot/snapshot_adapter_test.go

```diff
@@ -62,10 +62,8 @@ var _ = Describe("Snapshot Adapter", Ordered, func() {
 		deploymentTargetClass                     *applicationapiv1alpha1.DeploymentTargetClass
 		integrationTestScenario                   *v1beta1.IntegrationTestScenario
 		integrationTestScenarioWithoutEnv         *v1beta1.IntegrationTestScenario
```

Comment:
I guess, this and connected tests will be also removed in upcoming changes for 841.